src/test/java/com/algorand/algosdk/integration/Stepdefs.java

```diff
@@ -743,7 +743,7 @@ public void readTxn(String encodedTxn, String num) throws IOException {
         String path = System.getProperty("user.dir");
         Path p = Paths.get(path);
         this.num = num;
-        path = p.getParent() + "/temp/raw" + this.num + ".tx";
+        path = p.getParent() + "/temp.json/raw" + this.num + ".tx";
```

Comment:
Why make this change?

Reply:
Good catch, I am unsure why this change happen on my commit...